package.json

```diff
@@ -16,14 +16,15 @@
   "scripts": {
     "test-node": "mocha --recursive -R dot test/",
     "test-dev": "npm run test-node -- --watch -R min",
-    "test-headless": "mochify --recursive -R dot --grep WebWorker --invert test/",
+    "test-headless": "mochify --recursive -R dot --grep WebWorker --invert --plugin [proxyquire-universal] test/",
     "test-coverage": "mochify --recursive -R dot --grep WebWorker --invert --plugin [ mochify-istanbul --exclude '**/+(test|node_modules)/**/*' --report text --report lcovonly --dir ./coverage ] test/",
     "test-cloud": "npm run test-headless -- --wd",
     "test-webworker": "browserify --no-commondir --full-paths -p [ mocaccino -R spec --color ] test/webworker/webworker-support-assessment.js | phantomic --web-security false",
```

Comment:
Wasn't aware that adding `pre` was a general pattern outside of the standard lifecycle methods (publish, install, ...). You learn something every day :smile: Good improvement!

Having it as a dependency solves the grievances I had otherwise.